packages/mermaid/src/docs/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "vitepress --port 3333 --open",
+    "dev": "vitepress --port 3333 --host",
```

Comment:
But now the browser won't open for users no?

Reply:
Yep, it won't, they will have to click link in the console. Not very convenient. Now I have one more idea, perhaps we could add "dev:docker" script there

Reply:
Perhaps we could pass arguments down there somehow...

Reply:
> But now the browser won't open for users no?

Honestly, I wouldn't mind making this the default behavior for everybody.

I personally find it pretty annoying when I run `pnpm run dev` and it opens up a new tab.

Especially if you run `pnpm run dev` multiple times, you end up with having a bunch of tabs open for the same website, even though you only need one.

Reply:
I made a new command for that, it is a small duplication, the original one will be left intact